### PR TITLE
Make sure msgs for deleted numbers are failed

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
I have read the CLA Document and I hereby sign the CLA

File: backends/rapidpro/backend.go [hunk 1]
```diff
@@ -47,11 +47,11 @@ func init() {
 }
 
 // GetChannel returns the channel for the passed in type and UUID
-func (b *backend) GetChannel(ctx context.Context, ct courier.ChannelType, uuid courier.ChannelUUID) (courier.Channel, error) {
+func (b *backend) GetChannel(ctx context.Context, ct courier.ChannelType, uuid courier.ChannelUUID, isActive bool) (courier.Channel, error) {
 	timeout, cancel := context.WithTimeout(ctx, backendTimeout)
 	defer cancel()
 
-	return getChannel(timeout, b.db, ct, uuid)
+	return getChannel(timeout, b.db, ct, uuid, isActive)
```
Comment (rowanseymour) [inline backends/rapidpro/backend.go:54]: do we need to get deleted channels? Can't we just treat this as "courier can't find the channel, courier fails the message"

Comment (norkans7) [reply backends/rapidpro/backend.go:54]: I wanted to be sure that is for deleted channels only

Comment (norkans7) [reply backends/rapidpro/backend.go:54]: And we need a channel set to be able to write the status too

**Comment by rowanseymour** [comment]
Replaced by https://github.com/nyaruka/mailroom/pull/670

**Pull request closed without merging**